<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-10-06T18:54:13 IST" name="Default suite" finished-at="2023-10-06T18:55:02 IST" duration-ms="49150">
    <groups>
    </groups>
    <test started-at="2023-10-06T18:54:13 IST" name="Default test" finished-at="2023-10-06T18:55:02 IST" duration-ms="49150">
      <class name="com.GeneralStore.TestScripts.LoginToGeneralStore">
        <test-method is-config="true" signature="startServer()[pri:0, instance:com.GeneralStore.TestScripts.LoginToGeneralStore@3ce1e309]" started-at="2023-10-06T18:54:12 IST" name="startServer" finished-at="2023-10-06T18:54:13 IST" duration-ms="1355" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startServer -->
        <test-method is-config="true" signature="launchApp()[pri:0, instance:com.GeneralStore.TestScripts.LoginToGeneralStore@3ce1e309]" started-at="2023-10-06T18:54:13 IST" name="launchApp" finished-at="2023-10-06T18:54:39 IST" duration-ms="26006" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method signature="login()[pri:0, instance:com.GeneralStore.TestScripts.LoginToGeneralStore@3ce1e309]" started-at="2023-10-06T18:54:39 IST" name="login" finished-at="2023-10-06T18:54:57 IST" duration-ms="17751" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at com.GeneralStore.TestScripts.LoginToGeneralStore.login(LoginToGeneralStore.java:65)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="logoutFromApp()[pri:0, instance:com.GeneralStore.TestScripts.LoginToGeneralStore@3ce1e309]" started-at="2023-10-06T18:55:01 IST" name="logoutFromApp" finished-at="2023-10-06T18:55:02 IST" duration-ms="832" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromApp -->
        <test-method is-config="true" signature="stopServer()[pri:0, instance:com.GeneralStore.TestScripts.LoginToGeneralStore@3ce1e309]" started-at="2023-10-06T18:55:04 IST" name="stopServer" finished-at="2023-10-06T18:55:04 IST" duration-ms="97" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- stopServer -->
      </class> <!-- com.GeneralStore.TestScripts.LoginToGeneralStore -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
